Dockerfile.spdz

```
@@ -52,7 +52,8 @@ RUN git checkout ${ARTIFACT_TAG} && \
   git submodule set-url -- mpir https://github.com/wbhart/mpir
 
 RUN echo USE_GF2N_LONG = 0 >> CONFIG.mine && \
-  echo MY_CFLAGS = -DINSECURE >> CONFIG.mine
+  echo MY_CFLAGS = -DINSECURE >> CONFIG.mine && \
+  echo NO_CLIENT_TLS = 1 >> CONFIG.mine
```

That's also a compiler flag, so you'll need something like

```suggestion
  echo MY_CFLAGS = -DINSECURE -DNO_CLIENT_TLS >> CONFIG.mine && \
```

---

thank you!